docs/rst/introduction/introduction.rst

```diff
@@ -0,0 +1,71 @@
+.. _vulcanexus_global_introduction:
+
+Vulcanexus Overview
+===================
+
+*Vulcanexus*, the *ROS 2* all-in-one tool set, expands the *ROS 2* environment improving the developer experience with *ROS 2* providing open source features and tools not available with the current `ROS 2 release <https://www.ros.org/reps/rep-2000.html#release-schedule>`_.
+Consequently, *Vulcanexus* shares the underlying concepts, principles and architecture of :doc:`ROS 2 <../../ros2_documentation/source/index>`.
+For more information about *ROS 2* please refer to the `Robot Operating System 2: Design, architecture, and uses in the wild <https://www.science.org/doi/10.1126/scirobotics.abm6074>`_ [#]_.
+This section will follow and summarize the most important points contained in this paper, pointing out the differences and improvements that *Vulcanexus* provides.
+
+.. [#] S. Macenski, T. Foote, B. Gerkey, C. Lalancette and W. Woodall, "Robot Operating System 2: Design, architecture, and uses in the wild", Science Robotics vol. 7, May 2022
```

Comment:
I'd move this to the end of the page (footnote) as this page is short enough

Reply:
Done in 0d61e2fc8bfb6587e04c98238b18414c92b2c0b2